app/models/branch.rb

```diff
@@ -63,7 +63,7 @@ def pr_number
   end
 
   def pr_title_tags
-    pr_title.to_s.match(/^(?:\s*\[\w+\])+/).to_s.split.map { |s| s[1...-1] }
+    pr_title.to_s.match(/^(?:\s*\[\w+\])+/).to_s.gsub("[", " [").split.map { |s| s[1...-1] }
```

Comment:
yeah, it ends up not mattering because .split handles multiple consecutive spaces.  In fact, the other tests prove this as well.